src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,39 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static MAX_HEALTH = 100;
+
+  constructor(name) {
+    this.name = name;
+    this.health = Animal.MAX_HEALTH;
+    Animal.alive.push(this);
+  }
+  static alive = [];
 }
 
 class Herbivore extends Animal {
-  // write your code here
+  constructor(name) {
+    super(name);
+    this.hidden = false;
+  }
+
+  hide() {
+    this.hidden = true;
+  }
 }
 
 class Carnivore extends Animal {
-  // write your code here
+  bite(animal) {
+    const DAMAGE_AMOUNT = 50;
```

Comment:
Good, but `DAMAGE_AMOUNT` should be the property of `Carnivore` (not local variable inside the method)